sdk/src/rent.rs

```diff
@@ -28,8 +28,11 @@ pub const DEFAULT_EXEMPTION_THRESHOLD: f64 = 2.0;
 /// default amount of rent to burn, as a fraction of std::u8::MAX
 pub const DEFAULT_BURN_PERCENT: u8 = ((50usize * std::u8::MAX as usize) / 100usize) as u8;
 
-/// default base rent
-pub const DEFAULT_BASE_RENT_PER_YEAR: u64 = 112 * DEFAULT_LAMPORTS_PER_BYTE_YEAR;
+/// default account size (determined by size_of::<Account>())
```

Comment:
... at some ancient, pre-network time ;)  consider making this a power of 2, to make it look more like an arbitrary value?

Reply:
hmm. could round up to 128, I guess.

Reply:
👍

Reply:
Isn't 128 seems too high? I have set it to 32 currently.

Reply:
128 is good, IMHO, is closer to reality

Reply:
cool. Changed.